zer0-os/zOS
# feat: render user handle in conversation list header

### What does this do?
- renders user handle in conversation list header (i.e. primaryZID / wallet address / ' ').

### Why are we making this change?
- to display users primary zid if available. Alternatively render wallet address or nothing.

### How do I test this?
- for now you will only be able to render a users wallet address. login with a wallet and check the user handle in the user-header as shown below.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

Wallet
<img width="300" alt="Screenshot 2024-01-23 at 11 24 40" src="https://github.com/zer0-os/zOS/assets/39112648/4d796045-1b07-40d1-9724-0e1b7e4bc3a4">

ZID
<img width="300" alt="Screenshot 2024-01-23 at 11 24 54" src="https://github.com/zer0-os/zOS/assets/39112648/783f0293-0aa6-40d6-9253-e73ca04554f6">
